client/app/bundles/shared/components/Footer.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import css from './Footer.scss';
+
+const Footer = () => (
+  <div className={css.footer}>
+    <div id={css.footer_content}>
+      <div className={css.table}>
+        <div className={css.row}>
+          <div className={[css.table_cell, css.resources].join(' ')}>
+            <ul>
+              <h6 className={css.footer_header}>Resources</h6>
```

Comment:
We should be using i18n keys here since we translate our app into different languages.

@nshki @baohouse Do you think it's better to use i18n keys directly in here or pass the text through props? For a basic component like a button, I think passing through props makes sense. But a footer is a more "complicated" component 🤔

Reply:
I like passing the text through props as a consistent pattern across all components! It would keep the components a little cleaner and lower the barrier of entry for new contributors. That being said though, I don't have a lot of experience working with multilingual apps, so I'd love to know if there are any additional insights here I can learn from.

Reply:
Passing in a language mappings object into the Footer component via props just means you have yet-another-DTO to maintain, especially if I were using Flow or TypeScript. Keep it simple and just use directly. Libraries like react-18next automagically make it easy to use the direct-usage pattern, but under the hood is actually passing in an language map through the React component's context or props and rewriting the content of the <Trans> components.

Reply:
Actually on second thought, looking at the example at /apps/views/moments/index.html.erb, how would we directly use the ruby translation helper method? The React component is separate. We'll need to dig more into this.

Reply:
Hmmm it makes sense to use i18n directly in the component, but I see UI components as something reusable that you pass data into. We could split the footer component into smaller ones, have a generic component for each column. Then pass props into each one.